staging/src/k8s.io/code-generator/cmd/applyconfiguration-gen/args/args.go

```diff
@@ -57,7 +57,7 @@ func NewDefaults() (*args.GeneratorArgs, *CustomArgs) {
 	genericArgs.CustomArgs = customArgs
 
 	if pkg := codegenutil.CurrentPackage(); len(pkg) != 0 {
-		genericArgs.OutputPackagePath = path.Join(pkg, "pkg/client/applyconfigurations")
+		genericArgs.OutputPackagePath = filepath.Join(pkg, "pkg/client/applyconfigurations")
```

Comment:
a lot of the changes in the files in code-generator are computing packages, so should still use path, not filepath